package.json

```diff
@@ -20,6 +20,7 @@
     "react-scripts": "3.4.1",
     "sequelize": "^6.3.3",
     "sequelize-cli": "^6.2.0",
+    "sequelize-fixtures": "^1.2.0",
```

Comment:
I've added a library to make it easier to load test data, aka fixtures, into the db.